db/schema_tables.cc

```diff
@@ -2198,7 +2198,7 @@ std::vector<mutation> make_create_aggregate_mutations(schema_features features,
 
 std::vector<mutation> make_drop_aggregate_mutations(schema_features features, shared_ptr<cql3::functions::user_aggregate> aggregate, api::timestamp_type timestamp) {
     auto muts = make_drop_function_mutations(aggregates(), *aggregate, timestamp);
-    if (features.contains<schema_feature::SCYLLA_AGGREGATES>() && aggregate->has_reducefunc()) {
+    if (features.contains<schema_feature::SCYLLA_AGGREGATES>() && aggregate->is_reducible()) {
```

Comment:
Would have been less noisy if the original patch called it is_reducible().

Reply:
fixed